.ci/scripts/diff_dswx_files.py

```diff
@@ -0,0 +1,139 @@
+#   diff_dswx_files.py
```

Comment:
Since this is expected to be a callable script, the first line should be a [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) that instructs the OS on how the script should be executed.

For Python, this is `#!/usr/bin/env python3`